runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -476,7 +474,19 @@ else if (message.equals(BRACELET_OF_CLAY_USE_TEXT) || message.equals(BRACELET_OF
 				{
 					int charges = Ints.constrainToRange(getItemCharges(ItemChargeConfig.KEY_BRACELET_OF_CLAY) - 1, 0, MAX_BRACELET_OF_CLAY_CHARGES);
 					updateBraceletOfClayCharges(charges);
-					clayMinedCheck = true;
+				}
+			}
+			else if (message.equals(BRACELET_OF_CLAY_USE_TEXT_TRAHAEARN))
+			{
+				final ItemContainer equipment = client.getItemContainer(InventoryID.EQUIPMENT);
+				final ItemContainer inventory = client.getItemContainer(InventoryID.INVENTORY);
+
+				// Checks if player has twenty-seven items in inventory before taking away a charge
+				// The message runs before the item is received, so we can check if someone would receive one item
```

Comment:
![message_before_item](https://user-images.githubusercontent.com/104322242/201792914-d34fe5f9-07ad-4865-b8ae-5a51e9e64601.png)
Idk I see the message run before I receive an item in my personal testing

Reply:
Does `client.getItemContainer(INVENTORY).size()` differ at those two points? In my testing last night, it was 28 when the chat message was received, rather than 27.

Reply:
I'll work on getting an account up to where I can enter priff though in the meantime

Reply:
You can test this on any account with any ore, it wouldn't have to be the clay in trahaearn.

```java
@Subscribe
void onChatMessage(ChatMessage event)
{
	if (event.getType() == ChatMessageType.SPAM && event.getMessage().equals("Whatever the ore mined message is"))
	{
		log.debug("Inventory size at ChatMessage: {}", client.getItemContainer(ItemContainer.INVENTORY).size());
	}
}

@Subscribe
void onItemContainerChanged(ItemContainerChanged event)
{
	if (event.getContainerId() == ItemContainer.INVENTORY.getId())
	{
		log.debug("Inventory size at ItemContainerChanged: {}", event.getItemContainer().size());
	}
}
```